pkg/scheduler/scheduler.go

```diff
@@ -275,9 +267,10 @@ func (s *Scheduler) admit(ctx context.Context, e *entry) error {
 			log.V(2).Info("Workload not admitted because it was deleted")
 			return
 		}
+
 		log.Error(err, errCouldNotAdmitWL)
-		s.requeueAndUpdate(log, ctx, &e.Info, err.Error())
-		s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Pending", "%s: %v", errCouldNotAdmitWL, err)
+		e.inadmissibleReason = fmt.Sprintf("%s: %v", errCouldNotAdmitWL, err)
```

Comment:
I don't think we are using this value, are we?

Should we include it in the log?

Reply:
it may be not useful. I removed it.